src/test/java/uk/gov/hmcts/reform/coh/controller/EventForwardingControllerTest.java

```diff
@@ -1,64 +1,141 @@
+        SessionEventType sessionEventType = new SessionEventType();
+        sessionEventType.setEventTypeName("question_round_issued");
+        sessionEventType.setEventTypeId(5);
+        Jurisdiction jurisdiction = new Jurisdiction();
+        jurisdiction.setJurisdictionName("JUI");
+        jurisdiction.setJurisdictionId(2L);
+
+
+        given(sessionEventTypeService.retrieveEventType(any(String.class))).willReturn(Optional.of(sessionEventType));
+        given(jurisdictionService.getJurisdictionWithName(any(String.class))).willReturn(Optional.of(jurisdiction));
+
+        given(sessionEventForwardingRegisterService.retrieveEventForwardingRegister(
+                any(SessionEventForwardingRegister.class)))
+                .willReturn(Optional.empty());
+        mockMvc = MockMvcBuilders.standaloneSetup(eventForwardingController).build();
+    }
 
     @Test
-    public void testEventRegisteringWithJsonFile() throws Exception {
-        eventRegistrationRequest = (EventRegistrationRequest) JsonUtils.toObjectFromTestName("event_forwarding_register/valid_event_register", EventRegistrationRequest.class);
+    public void testCreateEventForwardRegister() throws Exception {
+
+        String json = JsonUtils.getJsonInput("event_forwarding_register/valid_event_register");
 
-        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT)
-                .contentType(MediaType.APPLICATION_JSON).content(JsonUtils.toJson(eventRegistrationRequest)))
+        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT+"/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(json))
                 .andExpect(status().is2xxSuccessful());
     }
 
     @Test
-    public void testMaximumRetriesDefault() throws Exception {
-        final Integer DEFAULT_RETRIES = 3;
+    public void testCreateEventForwardRegisterConflict() throws Exception {
 
-        eventRegistrationRequest = (EventRegistrationRequest) JsonUtils.toObjectFromTestName("event_forwarding_register/valid_event_register_no_retries", EventRegistrationRequest.class);
+        String json = JsonUtils.getJsonInput("event_forwarding_register/valid_event_register");
 
-        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT)
-                .contentType(MediaType.APPLICATION_JSON).content(JsonUtils.toJson(eventRegistrationRequest)))
-                .andExpect(status().is2xxSuccessful());
+        given(sessionEventForwardingRegisterService.retrieveEventForwardingRegister(
+                any(SessionEventForwardingRegister.class)))
+                .willReturn(Optional.of(new SessionEventForwardingRegister()));
 
-        assertEquals(DEFAULT_RETRIES, eventRegistrationRequest.getMaxRetries());
+        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT+"/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(json))
+                .andExpect(status().is4xxClientError());
+
+    }
+
+    @Test
```

Comment:
testCreateEventForwardRegisterMissingJurisdiction and testCreateEventForwardRegisterMissingEventType are the wrong war around